notebooks/official/prediction/llm_streaming_prediction.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
<p>No need to import the modules again "from vertexai.preview.language_models import TextGenerationModel</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
done